app/styles/join-section.module.css

```diff
@@ -0,0 +1,102 @@
+.join {
+  text-align: center;
+}
+
+.join__titleHighlighted {
+  color: var(--color-pinkl);
+  margin-top: -1.1rem;
+}
+
+.join__titleQuestion {
+  font-size: 3.3rem;
+}
+
+.join__para {
+  width: 52%;
+  text-align: justify;
+  margin: auto;
+  color: var(--color-darkblack);
+  font-size: 1.4rem;
+  font-weight: bold;
+  margin-top: 13px;
+}
+
+.button {
+  display: inline-block;
+  margin-top: 2.5rem;
+  text-decoration: none;
+  color: var(--color-white);
+  font-size: 1.3rem;
+  font-weight: bold;
+  border: none;
+  background-color: var(--color-navyblue);
+  border-radius: 0.8rem;
+  padding: 1rem 3rem;
+}
+
+.button:hover {
+  background-color: var(--color-pinkl);
+  transition-duration: 250ms;
+}
+
+@media (max-width: 1024px) {
+  .join__titleHighlighted {
+    font-size: 3.5rem;
+    margin-top: -0.9rem;
```

Comment:
I don't think we need a media query for the -ve margin, as per my observation it does not have much effect.